app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -75,7 +75,7 @@ const MakeDecision = ({ data }) => {
     startTime: Date.now(),
   });
   const [tabs, toggleTab, collapseTabsExceptCurr] = useAccordionTabsState({
-    [MACHINE_LEARNING_SUGGESTIONS]: isAnalyzerAvailable,
+    [MACHINE_LEARNING_SUGGESTIONS]: false,
```

Comment:
For now if you open modal window ML tab closed by default.(no loader or something yet)
If it is expected behavior we do not needed anymore loading flag and loader component in `machineLearningSuggestions` component.
Also now if data loading long time user can  open ML tab manually and see `ML has not found any suggestions`, but data is in loading process and it is bad user experience.